packages/adapter-static/package.json

```diff
@@ -17,6 +17,11 @@
 		"./package.json": "./package.json"
 	},
 	"types": "index.d.ts",
+	"files": [
+		"index.js",
+		"index.d.ts",
+		"platforms.js"
+	],
```

Comment:
Looks like we were publishing all files previously: https://www.npmjs.com/package/@sveltejs/adapter-static?activeTab=explore. I scoped this down to the ones we need.